site2/docs/admin-api-partitioned-topics.md

```diff
@@ -1,109 +1,82 @@
 ---
 id: admin-api-partitioned-topics
 title: Managing partitioned topics
-sidebar_label: Partitioned topics
+sidebar_label: Topics1
 ---
 
-
-You can use Pulsar's [admin API](admin-api-overview.md) to create and manage partitioned topics.
-
-In all of the instructions and commands below, the topic name structure is:
-
-```shell
-persistent://tenant/namespace/topic
-```
-
-## Partitioned topics resources
+## Manage partitioned topics
+You can use Pulsar [admin API](admin-api-overview.md) to create, update, delete and check status of partitioned topics.
 
 ### Create
 
-Partitioned topics in Pulsar must be explicitly created. When creating a new partitioned topic you
-need to provide a name for the topic as well as the desired number of partitions.
-
-> #### Note
->
-> By default, after 60 seconds of creation, topics are considered inactive and deleted automatically to prevent from generating trash data.
->
-> To disable this feature, set `brokerDeleteInactiveTopicsEnabled`  to `false`.
->
-> To change the frequency of checking inactive topics, set `brokerDeleteInactiveTopicsFrequencySeconds` to your desired value.
->
-> For more information about these two parameters, see [here](reference-configuration.md#broker).
+Partitioned topics must be explicitly created. When creating a new partitioned topic, you need to provide a name and the number of partitions for the topic.
 
-#### pulsar-admin
+By default, after 60 seconds of creation, topics are considered inactive and deleted automatically to prevent from generating trash data. To disable this feature, set `brokerDeleteInactiveTopicsEnabled` to `false`. To change the frequency of checking inactive topics, set `brokerDeleteInactiveTopicsFrequencySeconds` to your desired value.
```

Comment:
```suggestion
By default, 60 seconds after creation, topics are considered inactive and deleted automatically to avoid generating trash data. To disable this feature, set `brokerDeleteInactiveTopicsEnabled` to `false`. To change the frequency of checking inactive topics, set `brokerDeleteInactiveTopicsFrequencySeconds` as required.
```